.circleci/config.yml

```diff
@@ -420,22 +420,16 @@ jobs:
             FOUNDRY_PROFILE: ci
           working_directory: packages/contracts-bedrock
           no_output_timeout: 15m
-      - run:
-          name: storage snapshot
-          command: |
-            pnpm storage-snapshot
-            git diff --exit-code .storage-layout || echo "export STORAGE_SNAPSHOT_STATUS=1" >> "$BASH_ENV"
```

Comment:
Looks like there's still a check for this env var below that needs to be removed

Reply:
good catch, this has been addressed